src/ui/input/color-picker/color-picker.module.scss

```diff
@@ -0,0 +1,31 @@
+.label {
+  display: flex;
+  height: 40px;
+  position: relative;
+
+  input {
+    position: absolute;
+    top: 42px;
+  }
+}
+.indicator {
+  display: flex;
+
+  .textBox,
+  .colorBox {
+    border: 1px solid var(--bit-border-color-lightest, #ededed);
+    box-sizing: border-box;
+  }
+  .textBox {
+    border-right: none;
+    border-radius: 8px 0 0 8px;
+    display: flex;
+    align-items: center;
+    padding: 11px 16px;
+    font-size: 14px;
+  }
+  .colorBox {
+    width: 40px;
```

Comment:
hmm. we need to find a way to use it without setting a size.
We just want to keep a 1:1 aspect ratio, maybe there's a css trick for that?

Reply:
done